src/cel_python/runtime.py

```diff
@@ -1,48 +1,113 @@
-import sys
-from antlr4 import *
+import antlr4
 from .parser.CELLexer import CELLexer
 from .parser.CELParser import CELParser
-from .parser.CELListener import CELListener
-from .visitor_interp import VisitorInterp
+from .context import Context
+from .type_checker import TypeChecker
+from .interpreter import Interpreter
+from antlr4.error.ErrorListener import ErrorListener
+import logging
 
-class Runtime:
-    def __init__(self, cel_expression):
-        input_stream = InputStream(cel_expression)
-        lexer = CELLexer(input_stream)
-        token_stream = CommonTokenStream(lexer)
-        parser = CELParser(token_stream)
+# Configure logging
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+class ErrorCollector(ErrorListener):
```

Comment:
This class seems a duplicate from the one defined in `error_collector.py`.

Reply:
Removed!